network/controller/bootstrap/session_manager.go

```diff
@@ -62,7 +63,19 @@ type sessionWithID struct {
 
 type notification struct{}
 
-type SessionManager struct {
+type SessionManager interface {
+	component.Starter
+	// component.Stopper
```

Comment:
remove comment if not needed

Reply:
Stopper implementation is left commented till the bug with blocking on Stop is resolved